.github/workflows/ci.yml

```diff
@@ -26,7 +26,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [14, 16, 18, 19]
```

Comment:
Please retain Node 14, we are still supporting it with Fastify v4.x

Reply:
@Fdawgs Added Node 14 back :+1: